src/component/Animatable.lua

```diff
@@ -0,0 +1,16 @@
+--Animatable.lua
```

Comment:
Animatable acts a *lot* like a Renderable. What are your thoughts on combining Animatable and Renderable (or have a Renderable be reduced to a special-case 1-frame Animatable)?

In particular, consider this in the context of my comment on the RenderableSystem changes.